components/Dialogs/examples/DialogsCustomShadowExampleViewController.swift

```diff
@@ -54,30 +56,33 @@ class CustomShadowViewController: UIViewController {
 
 class DialogsCustomShadowExampleViewController: UIViewController {
 
-  let flatButton = MDCFlatButton()
+  let textButton = MDCButton()
   let transitionController = MDCDialogTransitionController()
+  var containerScheme = MDCContainerScheme()
 
   override func viewDidLoad() {
     super.viewDidLoad()
 
-    view.backgroundColor = UIColor.white
+    view.backgroundColor =
+        containerScheme.colorScheme?.backgroundColor ?? MDCSemanticColorScheme().backgroundColor
```

Comment:
Same comment as above.

Reply:
Resolving this so we can keep the conversation in one thread for all three of these.